src/nodes/typeGuards/index.ts

```diff
@@ -0,0 +1,4 @@
+export * from "./alias";
+export * from "./compound";
+export * from "./simple";
```

Comment:
[Nit] Although _"simple"_ is a good way to describe these, I try to avoid terms like it and _"complex"_ when referring to technical things. It can be off-putting for newcomers to be told something is simple when they don't yet understand it. 

Simplicity is also relative. One could argue that type guard functions are more complex than functions that return `boolean`.

How about ...

```suggestion
export * from "./single";
```